contrib/suil/update.py

```diff
@@ -0,0 +1,2 @@
+url = "https://drobilla.net/category/suil/"
+pattern = r"suil-([0-9.]+).tar.xz"
```

Comment:
```suggestion
pattern = r"suil-([\d.]+)\.tar\.xz"
```

ditto for the others

Reply:
I've applied that change, but we've got `[0-9.]` in 43 other `update.py` files. Should these all be changed as well to be consistent?